src/main/java/org/mabb/fontverter/opentype/CmapSubTable.java

```diff
@@ -186,12 +189,25 @@ public Format6SubTable() {
         public int glyphCount() {
             return 0;
         }
+        
+        @Override
+        public List<GlyphMapping> getGlyphMappings() {
+        	return GlyphMapReader.readCharCodesToGlyphs(charCodeToGlyphId, CFFStandardEncoding.getInstance());
+        }
 
         public void readData(FontDataInput input) throws IOException {
             int length = input.readUnsignedShort();
             rawReadData = input.readBytes(length - 4);
             input = new FontDataInputStream(rawReadData);
 
+            languageId = input.readUnsignedShort();
+            int firstCode = input.readUnsignedShort();
+            int entryCount = input.readUnsignedShort();
+            int[] glyphIndex = input.readUnsignedShortArray( entryCount );
+            
+            for( int i = 0; i < entryCount; i++ ) {
```

Comment:
```suggestion
            for(int i = 0; i < entryCount; i++) {
```

No spaces around the parenthesis